src/app/bug/shared/bug.service.spec.ts

```diff
@@ -1,11 +1,17 @@
 import { HttpClient } from '@angular/common/http';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
 import {  SpectatorHttp, createHttpFactory } from '@ngneat/spectator'
+import { BugDataService } from './bug-data.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8845963)
```suggestion
import { BugDataService } from "./bug-data.service";
```